packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMImageUtils.m

```diff
@@ -2,18 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import "FGMUtils.h"
-
-@interface FGMUtils ()
-
-@end
-
-@implementation FGMUtils
-
-+ (UIImage *)iconFromBitmap:(FGMPlatformBitmap *)platformBitmap
-                  registrar:(NSObject<FlutterPluginRegistrar> *)registrar
-                screenScale:(CGFloat)screenScale {
-  NSAssert(screenScale > 0, @"Screen scale must be greater than 0");
+#import "FGMImageUtils.h"
+#import "Foundation/Foundation.h"
+
+static UIImage * ScaleImage(UIImage *image, double scale);
```

Comment:
I think all of these methods should have lower case as a first letter: https://google.github.io/styleguide/objcguide.html#objective-c-method-names:~:text=Method%20and%20parameter%20names%20typically%20start%20as%20lowercase